pkg/kubelet/certificate/certificate_manager.go

```diff
@@ -170,7 +166,8 @@ func (m *manager) Current() *tls.Certificate {
 // CertificateSigningRequestClient that has already been set. This method is to
 // support the one specific scenario where the CertificateSigningRequestClient
 // uses the CertificateManager.
-func (m *manager) SetCertificateSigningRequestClient(certSigningRequestClient certificatesclient.CertificateSigningRequestInterface) error {
+func (m *manager) SetCertificateSigningRequestClient(
+	certSigningRequestClient certificatesclient.CertificateSigningRequestInterface) error {
```

Comment:
Bad fmt. Please keep it as it is.

Reply:
the line is too long, code convention does not prefer this